eventuals/grpc/client.h

```diff
@@ -378,7 +379,22 @@ class Client {
                  ::grpc::TemplatedGenericStub<
                      RequestType,
                      ResponseType>(channel_)},
-             callback = Callback<void(bool)>()](auto& k) mutable {
+             callback = Callback<void(bool)>()](auto& k, std::optional<Interrupt::Handler>& handler) mutable {
```

Comment:
You'll need to wrap this line, I'm ok for now with just doing the following if it's under 80 characters:
```suggestion
             callback = Callback<void(bool)>()](auto& k, auto& handler) mutable {
```